### ISSUE TITLE
styletron-client files are not minified by react-scripts

### ISSUE THREAD

**Issue by andreiduca** [rtsao/styletron#181]
In a React app with server-side rendering and rehydrated Styletron styles, the build process fails when minifying `styletron-client/dist/browser.es.js`.

Reduced test case:

1. create a new React app: `create-react-app test`
2. install Styletron packages: `yarn add styletron styletron-react styletron-client`
3. wrap the default `<App />` component in a Styletron provider:
```jsx
// ...
import StyletronClient from 'styletron-client';
import { StyletronProvider } from 'styletron-react';

// ...

const styleElements = document.getElementsByClassName('_styletron_hydrate_');
const styletronInstance = new StyletronClient(styleElements);

ReactDOM.render(
    <StyletronProvider styletron={styletronInstance}>
        <App />
    </StyletronProvider>,
    document.getElementById('root')
);
```
4. try to build the app with `yarn build` or `npm run build`
5. notice the build failing with the following error:
```sh
Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file: 

 	./node_modules/styletron-client/dist/browser.es.js:15 

Read more here: http://bit.ly/2tRViJ9
```

The link provided in the error message says that *"Some third-party packages don't compile their code to ES5 before publishing to npm."*, so it seems that some code is not correctly compiled to es5.

**Comment by rtsao** [comment]
Good catch! This should be resolved in `3.0.0-rc.3`.

**Comment by andreiduca** [comment]
I can confirm it works with the new release. Thank you!
